pages/builders-challenge/index.html

```diff
@@ -53,9 +53,15 @@
         { "name": "Algorithmic Fairness: A Pathway to Developing Responsible AI Systems", "desc": " - Join Golnoosh Farnadi, Canada CIFAR AI Chair, Mila, as she discusses the importance of fairness and provides an overview of techniques for ensuring algorithmic fairness through the machine learning pipeline while also suggesting questions and future directions for building a responsible AI system."},
         { "name": "Operationalizing AI Ethics", "desc": " - Dr. Karina Alexanyan, Director of Strategy at All Tech is Human, Katya Klinova, Head of AI, Labor and the Economy at Partnership on AI, Chris McClean, Director, Global Lead, Digital Ethics at Avanade, Gabriella de Queiroz, Principal Cloud Advocate at Microsoft, and Ehrik Aldana, Tech Policy Product Manager at Credo.ai will share insights and examples of how to be thoughtful and responsible about designing and deploying innovations in AI."},
         { "name": "Building Trust into Gen AI: Model Visibility and Tracking Change in Data Distributions", "desc": " - The current generation of Generative AI excels at working directly with humans in human language– and for several reasons this increases the potential for harm. Join Fiddler AI to generate a dataset by collaborating on a toy genAI task and explore techniques for localizing model weaknesses and time-dependent semantic drift."},
-        { "name": "Wait Wait, Don't push that button!” Human Centered Design for Responsible AI ", "desc": " - Join Superbloom for a workshop on human-centered design and AI, where we'll talk about designing for users in responsible ways that take into account challenges, benefits, and what can go wrong.s"},
-        { "time": "2:00pm - 3:20pm", "desc": "Keynote Speakers" },
+        { "name": "Wait Wait, Don't push that button!” Human Centered Design for Responsible AI ", "desc": " - Join Superbloom for a workshop on human-centered design and AI, where we'll talk about designing for users in responsible ways that take into account challenges, benefits, and what can go wrong."},
+        { "time": "2:00pm - 3:20pm", "desc": "Keynote Speakers"},
+        { "name": "Imo Udom", "desc": " - SVP Innovation Ecosystems"},
+        { "name": "Margaret Mitchell", "desc": " - Cheif Ethics Scientist, Hugging Face"},
```

Comment:
Typo: "Cheif" should be "Chief"
```suggestion
        { "name": "Margaret Mitchell", "desc": " - Chief Ethics Scientist, Hugging Face"},
```